src/proxy/ddl_test.go

```diff
@@ -80,6 +92,40 @@ func TestProxyDDLDB(t *testing.T) {
 	}
 }
 
+func TestProxyDDLDBError(t *testing.T) {
+	log := xlog.NewStdLog(xlog.Level(xlog.PANIC))
+	fakedbs, proxy, cleanup := MockProxy(log)
+	defer cleanup()
+	address := proxy.Address()
+
+	// create database querys
+	querysError := []string{
+		"create database test1 char set default",  // char-->charset
```

Comment:
Move some error which is parser error to the ERROR TEST on the sqlparser/ddl_test.go

Reply:
> Move some error which is parser error to the ERROR TEST on the sqlparser/ddl_test.go

I had move some errors to the sqlparser/ddl_test.go, and make the  comment more clear.

Reply:
Please Classify and classify by space，or just del the test moved to sqlparser/ddl_test.go